### PR TITLE
bnb-create-tables

### PR DESCRIPTION
**Pull request by Aya9292**
[commands]
- Authentication
[steps]
- login
- register
- forget password
[expected output]
- user can log in 
- user can register
- user can change the password
[note]

### CODE REVIEW

File: app/Http/Controllers/SocialAuthController.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+    class SocialAuthController extends Controller
```
Comment (rodenartuyo) [inline app/Http/Controllers/SocialAuthController.php]: Remove Indent before the class

File: database/migrations/2019_10_31_100521_create_rooms_table.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('rooms', function (Blueprint $table) {
+            $table->increments('id');
```
Comment (rodenartuyo) [inline database/migrations/2019_10_31_100521_create_rooms_table.php:17]: make all fields nullable

File: resources/views/auth/passwords/email.blade.php [hunk 3]
```diff
@@ -0,0 +1,45 @@
+@extends('layouts.app')
+
+@section('content')
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+
```
Comment (rodenartuyo) [inline resources/views/auth/passwords/email.blade.php]: Remove Spaces

File: app/Http/Controllers/SocialAuthController.php (hunk 1 above)
Comment (Aya9292) [reply app/Http/Controllers/SocialAuthController.php]: done

File: database/migrations/2019_10_31_100521_create_rooms_table.php (hunk 2 above)
Comment (Aya9292) [reply database/migrations/2019_10_31_100521_create_rooms_table.php:17]: arigato~done

File: resources/views/auth/passwords/email.blade.php (hunk 3 above)
Comment (Aya9292) [reply resources/views/auth/passwords/email.blade.php]: saramat ~~ done

**Review by rodenartuyo — APPROVED** [review]

**Pull request merged**